cockpit-project/cockpit
# eslint tweaks

- add eslint to `test/static-code`
 - make sure to catch our `#!/usr/bin/env node` scripts which don't necessarily end in `.js`
 - fix up a whole lot of problems
 - remove `eslint` from the webpack in production mode.  This is a substantial speedup.
